api/documentation/controller.py

```diff
@@ -738,6 +760,127 @@ def addSchemes(self):
+                            'description': 'A protocol name string'
+                        },
+                        'parent_id': {'type': 'string'}
+                    }
+                },
+                'protocols': {
+                    'type': 'object',
+                    'properties': {
+                        'name': {'type': 'string'},
+                        'label': {'type': 'string'},
+                        'description': {'type': 'string'},
+                        'settings': {
+                            'type': 'array',
+                            'items': {
+                                'type': 'object',
+                                'description': 'Mirror integration settings'
+                            }
+                        }
+                    }
+                }
+            }
+        )
+
+        self.addComponent(
+            'schema', 'CollectionsLibraryRegistrationPost', 'object',
+            {
+                'collection_id': {'type': 'string'},
+                'library_short_name': {'type': 'string'}
+            }
+        )
+
+        self.addComponent(
+            'schema', 'ServicesSchema', 'array',
+            {
+                'items': {
+                    'type': 'object',
+                    'properties': {
+                            'libraries': {
+                                'type': 'array',
+                                'items': {'type': 'string'}
+                            },
+                        'id': {'type': 'string'},
+                        'protocol': {'type': 'string'},
+                        'settings': {'type': 'object'},
+                    }
+                }
+            }
+        )
+
+        self.addComponent(
+            'schema', 'ServicesGetSchema', 'object',
+            {
+                '*_services': {'$ref': '#/components/schemas/ServicesSchema'
+                               },
+                'protocols': {'$ref': '#/components/schemas/ProtocolDictSchema'}
+            }
+        )
+
+        self.addComponent(
+            'schema', 'ServicesPostForm', 'object',
+            {
+                'protocol': {
+                    'type': 'string',
+                    'description': 'The name of a protocol to lookup'
+                },
+                'id': {'type': 'string'},
+                'name': {'type': 'string'},
+                'url': {'type': 'string'}
+            }
+        )
+
+        self.addComponent(
+            'schema', 'WorkComplaintsSchema', 'object',
+            {
+                'book': {
+                    'type': 'object',
+                    'properties': {
+                        'identifier_type': {'type': 'string'},
+                        'identifier': {'type': 'string'}
+                    }
+                },
+                'complaints': {
+                    'type': 'object',
+                    'description': 'A counter object of complaint types for a given work'
```

Comment:
Is it possible what the counter object looks like? Or is that not obvious from the codebase? Knowing what kind of types of complaints can be here would be helpful.